src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputParameterConverter.cs

```diff
@@ -123,7 +123,8 @@ public static InputParameter CreateInputParameter(ref Utf8JsonReader reader, str
 
         private static SerializationFormat GetSerializationFormat(InputType parameterType, RequestLocation requestLocation)
         {
-            if (parameterType is InputPrimitiveType { Kind: InputTypeKind.DateTime })
+            var affectType = parameterType is InputListType listType? listType.ElementType : parameterType;
```

Comment:
should we do the same for dictionary?
And this looks very much like a workaround

Reply:
Yes. it is a workaround now. It should be handled in `SerializationBuilder.GetSerializationFormat`. After we migrate to TCGC client interface, we don't need to handle DateTime default format according to the location (path, qurery or body), we can remove all this converter.